letsencrypt/cli.py

```diff
@@ -850,7 +876,7 @@ def _reconstitute(config, full_path):
     # webroot_map is, uniquely, a dict, and the general-purpose
     # configuration restoring logic is not able to correctly parse it
     # from the serialized form.
-    if "webroot_map" in renewalparams:
+    if "webroot_map" in renewalparams and not _set_by_cli("webroot_map"):
```

Comment:
It's safe to pass it on because its default value is `{}`, which is false.  If it's set by the user, it's likely to become true.